tensorflow/core/api_def/base_api/api_def_MatrixTriangularSolve.pbtxt

```diff
@@ -53,5 +52,34 @@ The output is a tensor of shape `[..., M, K]`. If `adjoint` is
 If `adjoint` is `False` then the strictly then the  innermost matrices in
 `output` satisfy matrix equations
 `adjoint(matrix[..., i, k]) * output[..., k, j] = rhs[..., i, j]`.
+
+Example:
+```python
+
+a = tf.constant([[3,  0,  0,  0],
+                 [2,  1,  0,  0],
+                 [1,  0,  1,  0],
+                 [1,  1,  1,  1]], dtype=tf.float32)
+
+b = tf.constant([[4],
+                 [2],
+                 [4],
+                 [2]], dtype=tf.float32)
+
+x = tf.linalg.triangular_solve(a, b, lower=True)
+x
+# <tf.Tensor: id=257, shape=(4, 1), dtype=float32, numpy=
+# array([[ 1.3333334 ],
+#        [-0.66666675],
+#        [ 2.6666665 ],
+#        [-1.3333331 ]], dtype=float32)>
+
+a@x
```

Comment:
Does this work? It doesn't for me.

Reply:
https://colab.research.google.com/drive/1e_7Xvmq1eHCcfL4W5lJn7vLRRUVfuWC1#scrollTo=BEvr6sT22nN_

do you mean, that for 1.13 I would need to enable eager? 
Just tested and it works for `1.13` as well as for `2.0.0-dev20190422`

Reply:
ah, you use python 2.7, right? both of my envs are python3

Reply:
@martinwicke Thanks for review! Updated